javascript/HtmlEditorField.js

```diff
@@ -868,14 +868,15 @@ ss.editorWrappers['default'] = ss.editorWrappers.tinyMCE;
 						// 	// Element contains a link
 						// 	var firstLinkEl = selectedEl.find('a:first');
 						// 	if(firstLinkEl.length) linkDataSource = firstLinkEl;
+
+                        // if(linkDataSource && linkDataSource.length) this.modifySelection(function(ed){
```

Comment:
I would normally not add such code but I feel in this case it should be done. It is because this code is related to a currently disabled feature of the HTML editor. The feature is adjustment of the selection in the case of overselection (user selects more than just the `anchor` element). This feature would reduce the selection to just the anchor.

It is currently disabled because of some technical issues that are documented in the code. The commented code is relevant in case the feature gets repaired and enabled.